### DISCUSSION TITLE
JavaScript ecmascript 5 ecmascript 6

### DISCUSSION THREAD

**Discussion by kban55** [codewars/codewars.com#2737] [Q&A]
Hi guys! :)

I need help. I'd like to learn JavaScript on codewars portal but not version ecmascript 6 but ecmascript 5 and I don't know where I can change it to ecmascript 5.

Can someone help me?
![ecma6](https://user-images.githubusercontent.com/94796232/176269411-ba7dbeb3-8b3c-42a3-844c-1445f84c3499.PNG)

**Comment by nikhilstacks** [comment]
you can use es5 in es6 you don't have to change it. only change the syntax back to function from variable assignment with arrow function.
change to:

function zeroFuel(sistanceToPump, mpg, fuelLeft){
//Todo
}

this will work fine.
Try to learn es6 so that you can cope up.

**Reply by ghulamrabbani883** [reply]
It will work but if there are more than one arrow function and variables defined using let and const, it will create problem due to hoisting issue. You need to keep track of the functions definition and calling

**Reply by nikhilstacks** [reply]
In code wars platform there is only one function in every question which we have to extend to have our answers. We don't need to change several functions to this format.

**Reply by ghulamrabbani883** [reply]
I am new to code wars, and may be we need helper function

**Reply by nikhilstacks** [reply]
we don't need to write extra functions, all we need is to learn es6 which is not that hard it's just syntactical sugar or we can convert them to normal functions.

**Reply by kban55** [reply]
Thank you for all replies. I guess that I should also learn ecmascript 6 then.
